libs/payments/ui/src/lib/server/purchase-details.tsx

```diff
@@ -5,11 +5,57 @@ import {
   getLocalizedCurrency,
   getLocalizedCurrencyString,
 } from '@fxa/shared/l10n/fluent';
+import { FluentBundle } from '@fluent/bundle';
 import { headers } from 'next/headers';
 import Image from 'next/image';
 import { formatPlanPricing } from '../utils/helpers';
+import '../../styles/index.css';
```

Comment:
Since this component is making use of styles provided by the component library, make sure to import those styles.